Assets/Scripts/Health.cs

```diff
@@ -9,20 +10,23 @@ public class Health : MonoBehaviour
     public Image screenBar;
     public Text percetengeBox;
     private SphereCollider earthCollider;
-
     public GameObject explossionPrefab;
     public GameObject fog;
-    GameObject geo;
-
     public int earthHealth;
     public int currentHealth;
     public float mHealth;
     public float mPercentage;
+    public AppController AppController;
```

Comment:
Keep it in here since it works. 

But for future, it is best for Health component to not know about AppController. Tightly coupling it breaks the point of a component.